Library/Formula/viewvc.rb

```diff
@@ -0,0 +1,36 @@
+class Viewvc < Formula
+  desc "Browser interface for CVS and Subversion repositories"
+  homepage "http://www.viewvc.org"
+  url "http://viewvc.tigris.org/files/documents/3330/49392/viewvc-1.1.23.tar.gz"
+  sha256 "d14290b23f44e8f5301a7ca9fc946bcd0f8bb8206d9b9c111ac56e19f2588b7d"
+
+  # swig is a dependency of subversion --with-python, but due to a
+  # bug it needs to also be specified here.
+  # https://github.com/Homebrew/homebrew/issues/42915
+  depends_on "swig" => :run
+  depends_on "subversion" => "with-python"
+
+  def install
+    system "/usr/bin/python", "./viewvc-install", "--prefix=#{libexec}", "--destdir="
```

Comment:
It'll use the system Python regardless unless people install with `--env=std`, if that was the concern behind this particular line. Superenv should prevent Homebrew's Python from wandering into the `$PATH` during normal installs.